README.md

```diff
@@ -41,6 +41,8 @@ As an alternative to the section above, you can run the app within a Docker cont
 
 * `$ docker run -p 5000:5000 -e "GITHUB_TOKEN=YOUR_TOKEN" hacktoberfest-checker`
 
+* `$ docker-compose up -d --build`
```

Comment:
I don't know too much about docker tbh, but presumably this is an alternative to running the run command above? If that is the case can we add an `or..` or something just above this?